### DISCUSSION TITLE
Function keys above 20 are not registered by scriptkit

### DISCUSSION THREAD

**Discussion by hakubo** [johnlindquist/kit#967] [Errors] [answered]
I have a programmable keyboard so I set a lot of shortcuts for Fxx but when I got to F21 - it appears that script kit does not register it.

I was able to verify that system do see it via Karabiner Event viewer.

**Comment by johnlindquist** [comment, marked as answer]
I love this question, hehe.

It seems like Karabiner has lower-level access to the system to listen for what we'll call "special keys" than the majority of apps do. I hope to be able to address this "someday" through some low-level custom native code, but it won't happen for a long, long time.
